corehq/apps/importer/tests.py

```diff
@@ -272,7 +272,7 @@ def testExternalIdMatching(self):
     def test_external_id_matching_on_create_with_custom_column_name(self):
         headers = ['id_column', 'age', 'sex', 'location']
         external_id = 'external-id-test'
-        config = self._config(headers, search_field='external_id')
+        config = self._config(headers[1:], search_column='id_column', search_field='external_id')
```

Comment:
the config object is really weird but by default it doesn't load whatever column you specify as the `external_id` so it won't actually be in the configured list of columns which was causing the failure